### PR TITLE
Ability make events in each day from start to end

### PR DESCRIPTION
**Pull request by petrukhinilya**

### CODE REVIEW

File: ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js [hunk 1]
```diff
@@ -6,25 +6,28 @@ export default function createCalendar(day, getallEvents, dayStyles, setValue) {
...
-              }
-            </div>
-          </div>
+    <div className='day' onClick={() => setValue(day)} >
+      <div id='number'>
+        <div className={dayStyles(day)}>{day.format('D').toString()}
+        </div>
+        <div>
+          {allEvents.map((event) => {
+            // console.log(event)
```
Comment (d-lazarev) [inline ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js]: The consoles and commented code shouldn't be here
Also, if you don't do any modifications with data in the `map` callback, you can don't write a `return` word and just render with a round bracket

File: ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js [hunk 2]
```diff
@@ -6,25 +6,28 @@ export default function createCalendar(day, getallEvents, dayStyles, setValue) {
...
-                    <div className='highlight'>{event.event}</div>
-                  )
-                })
-              }
-            </div>
-          </div>
+    <div className='day' onClick={() => setValue(day)} >
+      <div id='number'>
+        <div className={dayStyles(day)}>{day.format('D').toString()}
+        </div>
```
Comment (d-lazarev) [inline ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js]: fix codestyle

File: ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js [hunk 3]
```diff
@@ -6,25 +6,28 @@ export default function createCalendar(day, getallEvents, dayStyles, setValue) {
...
+        <div>
+          {allEvents.map((event) => {
+            // console.log(event)
+            return (
+              <div className='highlight'>{event.event}</div>
+            )
+          })
+          }
```
Comment (d-lazarev) [inline ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js]: fix codestyle

**Review by d-lazarev — CHANGES_REQUESTED** [review]
Please, see my notes below

File: ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js (hunk 1 above)
Comment (petrukhinilya) [reply ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js]: I belive that word Return will help me in future. I will modificate my data later.
Fixed

File: ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js (hunk 3 above)
Comment (petrukhinilya) [reply ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js]: Fixed

File: ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js (hunk 2 above)
Comment (petrukhinilya) [reply ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js]: Fixed, made divs in one line

File: ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js (hunk 3 above)
Comment (d-lazarev) [reply ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js]: Was fixed not completely

Fixed , in next PR

**Review by d-lazarev — CHANGES_REQUESTED** [review]
See my notes

File: ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js [hunk 4]
```diff
@@ -17,12 +17,25 @@ export default function createCalendar(day, getallEvents, dayStyles, setValue) {
...
         <div className={dayStyles(day)}>{day.format('D').toString()}</div>
+        <div>
           {allEvents.map((event) => {
+              const deleteEvents = () => {
+                deleteEvent(event._id);
+              }
+              const updateEvents = () => {
+                setEvent(event);
+                setChangePopup(true);
+              }
```
Comment (d-lazarev) [inline ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js]: Why these functions are presented in the `map` callback?

File: ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js (hunk 3 above)
Comment (d-lazarev) [reply ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js]: Should be fixed in this PR

**Review by d-lazarev — CHANGES_REQUESTED** [review]
Notes

File: ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js (hunk 4 above)
Comment (petrukhinilya) [reply ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js]: When switched branch these files was added automatically

Comment (petrukhinilya) [reply ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js]: I have a lot of mistakes if relocate this functions cause funcs interacts this 'event' inside map

File: ProjectCalendar/Frontend/src/Components/Calendar/ChangePopup.js [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+            },
+            endDate: {
+                value: endDate
+            },
+            text: {
+                value: text
+            },
+        }
+      }
+    } = e;
```
Comment (d-lazarev) [inline ProjectCalendar/Frontend/src/Components/Calendar/ChangePopup.js:34]: 20 - 34  - Fix alignment

File: ProjectCalendar/Frontend/src/Components/Calendar/ChangePopup.js [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+        break;
+    default:
+        break;
+    }
+  }
+
+  return (
+      <div className='main'>
+          <div className='wrapper-popup' onClick={onClick}>
+          </div>
```
Comment (d-lazarev) [inline ProjectCalendar/Frontend/src/Components/Calendar/ChangePopup.js]: Fix Alignment

File: ProjectCalendar/Frontend/src/Components/Calendar/ChangePopup.js [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+        <div>
+          <form className="popup" onSubmit={updateEvent}>
+            <input type='date' className='input1' onChange={onChange} value={startDate} name="startDate"></input>
+            <input type='date' className='input2' onChange={onChange} value={endDate} name="endDate"></input>
+            <input type='text' className='text' placeholder='Add event to date' onChange={onChange} value={text} name="text"></input>
+            <button type='submit' className='add-btn'>Change this event</button>
+            <input type='reset' onClick={onClick} className='reset'></input>
```
Comment (d-lazarev) [inline ProjectCalendar/Frontend/src/Components/Calendar/ChangePopup.js]: Use autoclosing for tags that don't have the body

File: ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js [hunk 8]
```diff
@@ -1,30 +1,35 @@
...
-                })
-              }
-            </div>
-          </div>
+    <div className='day' onClick={() => setValue(day)} >
+      <div id='number'>
+        <div className={dayStyles(day)}>{day.format('D').toString()}</div>
+        <div>
+          {allEvents.map((event) => {
+
```
Comment (d-lazarev) [inline ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js]: Unnecessary to skip a line

File: ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js [hunk 9]
```diff
@@ -1,30 +1,35 @@
...
+            return (
+              <div className='highlight'>
+                <div className = 'event' >{event.event}</div>
+                <button className='del' >Del</button>
```
Comment (d-lazarev) [inline ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js]: Fix codestyle

File: ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js (hunk 3 above)
Comment (d-lazarev) [reply ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js]: Still wasn't fixed

File: ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js [hunk 10]
```diff
@@ -1,30 +1,35 @@
...
-            <div>
-              {allEvents.map((event) => {
-                  return (
-                    <div className='highlight'>{event.event}</div>
-                  )
-                })
-              }
-            </div>
-          </div>
+    <div className='day' onClick={() => setValue(day)} >
```
Comment (d-lazarev) [inline ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js]: fix codestyle

**Review by d-lazarev — CHANGES_REQUESTED** [review]
Notes

File: ProjectCalendar/Frontend/src/Components/Calendar/ChangePopup.js (hunk 5 above)
Comment (petrukhinilya) [reply ProjectCalendar/Frontend/src/Components/Calendar/ChangePopup.js:34]: Fixed

File: ProjectCalendar/Frontend/src/Components/Calendar/ChangePopup.js (hunk 7 above)
Comment (petrukhinilya) [reply ProjectCalendar/Frontend/src/Components/Calendar/ChangePopup.js]: fixed

File: ProjectCalendar/Frontend/src/Components/Calendar/ChangePopup.js (hunk 6 above)
Comment (petrukhinilya) [reply ProjectCalendar/Frontend/src/Components/Calendar/ChangePopup.js]: Resolved

File: ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js (hunk 8 above)
Comment (petrukhinilya) [reply ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js]: Fixed

File: ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js (hunk 9 above)
Comment (petrukhinilya) [reply ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js]: Fixed

File: ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js (hunk 10 above)
Comment (petrukhinilya) [reply ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js]: Fixed

File: ProjectCalendar/Frontend/src/Components/Calendar/ChangePopup.js [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+            <input type='date' className='input2' onChange={onChange} value={endDate} name="endDate"/>
+            <input type='text' className='text' placeholder='Add event to date' onChange={onChange} value={text} name="text"/>
+            <button type='submit' className='add-btn'>Change this event</button>
+            <input type='reset' onClick={onClick} className='reset'/>
+          </form>
+        </div>
+      </div>
+    )
+}
+export default ChangePopup;
```
Comment (d-lazarev) [inline ProjectCalendar/Frontend/src/Components/Calendar/ChangePopup.js]: No newline at end of file

File: ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js (hunk 3 above)
Comment (petrukhinilya) [reply ProjectCalendar/Frontend/src/Components/Calendar/CreateCalendar.js]: Fixed

**Review by d-lazarev — APPROVED** [review]

**Pull request merged**